### ISSUE TITLE
k3OS always shuts down machine after successful PXE install

### ISSUE THREAD

**Issue by stellirin** [rancher/k3os#339]
**Version**

`v0.8.0`

**Architecture**

`amd64`

**Describe the bug**

k3OS always shuts down the machine after PXE install, even ignoring `k3os.install.power_off=false`

**To Reproduce**

PXE boot with something like:
```
LABEL k3os-server
    MENU LABEL K3OS Server
    KERNEL k3os/v0.8.0/k3os-vmlinuz-amd64
    INITRD k3os/v0.8.0/k3os-initrd-amd64
    APPEND k3os.mode=install k3os.install.iso_url=http://server/k3os/v0.8.0/k3os-amd64.iso k3os.install.config_url=http://server/k3os/k3os-server.yaml k3os.install.device=/dev/sda k3os.install.power_off=false
```

**Expected behavior**

Machine reboots directly into k3OS.

**Actual behavior**

Machine shuts down after successful install.

**Additional context**

It is not any strange behaviour of my hardware, I explicitly see 'shutting down' flash on the terminal after install is finished.

**Comment by dweomer** [comment]
Possibly addressed by #143. Need to test.

**Comment by amiga23** [comment]
Works for me with 0.9.0-rc2

**Comment by dweomer** [comment]
Appears to have been fixed by #143
